eng/pipelines/coreclr/templates/build-job.yml

```diff
@@ -109,6 +109,11 @@ jobs:
     - ${{ if and(eq(parameters.buildConfig, 'Release'), and(eq(parameters.osGroup, 'Windows_NT'), not(or(eq(parameters.archType, 'arm64'), eq(parameters.archType, 'arm'))))) }}:
       - name: enforcePgoArg
         value: '-enforcepgo'
+    - name: publishLogsArtifactPrefix
+      value: 'BuildLogs_CoreCLR'
+    - ${{ if eq(parameters.useGCC, true) }}:
+      - name: publishLogsArtifactPrefix
```

Comment:
Maybe we could just move this to the existing if that probes for `useGCC==true`?

Reply:
Merged into existing branch.